### DISCUSSION TITLE
Patch for IE9 and 10 support

### DISCUSSION THREAD

**Discussion by dr-kd** [alpinejs/alpine#671] [4. General]
Diff attached.  We've been using this in production for a while against version 2.2.0
[ie9.txt](https://github.com/alpinejs/alpine/files/4963486/ie9.txt)

**Comment by ryangjchandler** [comment]
@singingfish - thanks for sharing. I'm sure you can understand why Alpine doesn't support IE9/10, especially since they're so old and really, really shouldn't ever be used by anyone.

**Comment by dr-kd** [comment]
Unfortunately is the reality is that some customers refuse to update their infrastructure, and while < IE9 is no longer really possible due to SSL support, IE<11 has still got significant share in the "enterprise" space.
